lib/pay/paddle.rb

```diff
@@ -35,6 +35,10 @@ def self.environment
       find_value_by_name(:paddle, :environment) || "production"
     end
 
+    def self.public_key_file
+      find_value_by_name(:paddle, :public_key_file)
```

Comment:
This does read a file at https://github.com/pay-rails/pay/blob/d0b988e78a0587d99e08f8cc48f69f73c785d0ba/lib/pay/paddle/webhooks/signature_verifier.rb#L17 which is why I thought it would be best to name it as file

Reply:
Oh, I was more thinking of it just being a String since that way you could put it in your Rails credentials or secrets.